.gitignore

```diff
@@ -54,3 +54,7 @@ README.html
 sandbox
 
 !tests/.env
+
+# IDEA
+
+/.idea/
```

Comment:
This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
I reverted that one.